metas.groovy

```diff
@@ -0,0 +1,137 @@
+//Adapted from module.groovy
+@GrabResolver(name = 'jcenter', root = 'http://jcenter.bintray.com/')
+@Grab(group='org.ajoberstar', module='grgit', version='1.9.3')
+import org.ajoberstar.grgit.Grgit
+import groovy.json.JsonSlurper
+
+Properties properties = new Properties()
+new File("gradle.properties").withInputStream {
+    properties.load(it)
+}
+
+githubHome = properties.alternativeGithubHome ?: "MetaTerasology"
+
+modulesRetrieved = []
+
+
+def retrieve(String[] modules) {
+    println "Now inside retrieve, user wants: $modules"
+    for (String module : modules) {
+        println "Starting loop for meta module $module"
+        println "Meta Modules retrieved so far: $modulesRetrieved"
+        retrieveModule(module)
+    }
+}
+
+def retrieveModule(String module) {
+    File targetDir = new File("meta/$module")
+    println "Request to retrieve meta module $module would store it at $targetDir - exists? " + targetDir.exists()
+
+    if (targetDir.exists()) {
+        println "That meta module already had an existing directory locally. If something is wrong with it please delete and try again"
+        modulesRetrieved << module
+    } else if (modulesRetrieved.contains(module)) {
+        println "We already retrieved $module - skipping"
+    } else {
+        println "Retrieving meta module $module - if it doesn't appear to exist (typo for instance) you'll get an auth prompt (in case it is private)"
+        //noinspection GroovyAssignabilityCheck - GrGit has its own .clone but a warning gets issued for Object.clone
+        Grgit.clone dir: targetDir, uri: "https://github.com/$githubHome/${module}.git"
+        modulesRetrieved << module
+
+        // TODO: Temporary until build.gradle gets removed from module directories (pending Cervator work from module.groovy)

```

Comment:
This can be removed (the meta modules will never have a `build.gradle`)

Reply:
OK Sir!